Cargo.toml

```diff
@@ -20,6 +20,7 @@ log = "0.4.14"
 env_logger = "0.9.0"
 lazy_static = "1.4.0"
 string-error = "0.1.0"
+rust-crypto = "^0.2"
```

Comment:
Hi @voidC1326, I work in s390x Kata development
Two things from [rust-crypto's README](https://github.com/DaGenix/rust-crypto/) worry me a bit:
> The x86-64, x86, and ARM architectures are supported

Despite this code compiling on s390x, support for s390x (and ppc64le) would be important.
> Rust-Crypto has not been thoroughly audited for correctness, so any **use where security is important is not recommended** at this time.

(emphasis mine)

The (already used) [aes-gcm](https://github.com/RustCrypto/AEADs/tree/master/aes-gcm) has neither of these issues.
Maybe this isn't a showstopper since it only appears to be used in the sample KBC at this time. But I don't think the sample should use something any practical implementation shouldn't.